lib/workers/global/index.ts

```diff
@@ -103,7 +102,6 @@ export async function start(): Promise<number> {
         break;
       }
       const repoConfig = await getRepositoryConfig(config, repository);
```

Comment:
```suggestion
      const repoConfig = await getRepositoryConfig(config, repository);
      await removeDanglingContainers();
```

Shouldn't we call it here, so it's done only once? @rarkins What do you think?

Reply:
Sure, that would seem most backwards-compatible

Reply:
Should be called after `setAdminConfig()`, as we're querying `binarySource`

Reply:
ok, so admin config should be set only once too, as it can't change per repo.

Reply:
Maybe do it later with other PR? It may require more changes of how we handle `localDir` which seems to be the only field being set per-repo.